internal/provider/resource_gitlab_group_test.go

```diff
@@ -275,6 +275,63 @@ func TestAccGitlabGroup_PreventForkingOutsideGroup(t *testing.T) {
 	})
 }
 
+func TestAccGitlabGroup_updatedAttributes(t *testing.T) {
+	var group gitlab.Group
+	var project gitlab.Project
+	rInt := acctest.RandInt()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: providerFactories,
+		CheckDestroy:      testAccCheckGitlabGroupDestroy,
+		Steps: []resource.TestStep{
+			{
+				//This test requires EE because without it, the two "Shared Runner" settings will always return 0,
+				//  even though they will work. They also require an admin token.
+				SkipFunc: isRunningInCE,
+				Config:   testAccGitLabGroupWithProjectTemplate(rInt),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckGitlabGroupExists("gitlab_group.test-group", &group),
+					testAccCheckGitlabProjectExists("gitlab_project.foo", &project),
+					resource.TestCheckResourceAttr("gitlab_group.test-group",
+						"file_template_project_id", fmt.Sprintf("%d", project.ID)),
+				),
+			},
```

Comment:
With a subsequent import verification step you'd verify that not only the terraform state is correctly set, but also the upstream API is correctly called.

Reply:
I'm working on adding this, but I'm getting some intermittent failures when running tests locally that I'm not happy with, so I'm doing some debugging.

Reply:
So, turns out there are some weird debugging quirks with GitPod that I'll see if I can figure out in a different issue. Swapping to local development resolved the intermittent failures. 

I updated the comment line because the test still does require EE I believe (otherwise the file_template_project_id just doesn't return), and added an import test.